packages/console/src/consts/subscriptions.ts

```diff
@@ -1,17 +1,9 @@
 import { ReservedPlanId } from '@logto/schemas';
-import { condArray } from '@silverhand/essentials';
-
-import { isDevFeaturesEnabled as isDevelopmentFeaturesEnabled } from './env';
 
 /**
  * In console, only featured plans are shown in the plan selection component.
  */
-export const featuredPlanIds: string[] = [
-  ReservedPlanId.Free,
-  ReservedPlanId.Hobby,
-  // Todo @xiaoyijun [Pricing] Remove feature flag
-  ...condArray(!isDevelopmentFeaturesEnabled && ReservedPlanId.Pro),
-];
+export const featuredPlanIds: string[] = [ReservedPlanId.Free, ReservedPlanId.Hobby];
```

Comment:
Why do we remove Pro instead of Hobby?

Reply:
Since now the `hobby` plan is the new pro plan.